client/src/components/JsonViewer.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <Json v-for="(item, key) in json" :name="key" :value="item" :open="true" />
```

Comment:
Gleiches hier
Elements in iteration expect to have 'v-bind:key' directives.eslint-plugin-vue